README.md

```diff
@@ -80,9 +80,9 @@ Train your mine locally using a sonar smart contract
 
 Options:
 
-  -m, --mine-address <hexstring>      Blockchain address for the mine to use
+  -m, --mine-address <hexstring or auto>      Blockchain address for the mine to use. `auto`` sets the mine to pick a random account.
```

Comment:
Also: We might just remove this static help from the readme and refer to the `mine help` option?